rct_optimizations/test/pnp_utest.cpp

```diff
@@ -159,7 +159,7 @@ TEST_F(PnP3DTest, PerturbedInitialCondition)
   PnPResult result = optimize(problem);
   EXPECT_TRUE(result.converged);
   EXPECT_TRUE(result.camera_to_target.isApprox(target_to_camera.inverse(), 1.0e-6));
-  EXPECT_LT(result.final_cost_per_obs, 1.0e-12);
+  EXPECT_LT(result.final_cost_per_obs, 1.0e-10);
```

Comment:
I think we need a better way to evaluate these variable costs in our tests. Maybe we should compare costs across multiple optimizations and check that the mean and variance of all the costs are within some specified range.